agent/consul/server.go

```diff
@@ -503,6 +516,33 @@ func NewServerLogger(config *Config, logger *log.Logger, tokens *token.Store, tl
 	return s, nil
 }
 
+func (s *Server) trackAutoEncryptCARoots() {
+	for {
+		select {
+		case <-s.shutdownCh:
+			s.logger.Printf("[DEBUG] agent: shutting down trackAutoEncryptCARoots because shutdown")
```

Comment:
This could be simplified to be more consistent with errors like `ErrSessionShutdown`: 
```suggestion
			s.logger.Printf("[DEBUG] agent: trackAutoEncryptCARoots shutdown")
```